CHANGELOG.md

```diff
@@ -23,6 +24,7 @@ Breaking changes:
   - `Source Control` and `Explorer` are view containers now and previous layout data cannot be loaded for them. Because of it the layout is completely reset.
 - [vscode] complete support of variable substitution [#5835](https://github.com/theia-ide/theia/pull/5835)
   - inline `VariableQuickOpenItem`
+- [core] refactored `TreeDecoration` to `WidgetDecoration` and moved it to shell, since it is a generic decoration that can be used by different types of nodes (currently by tree nodes and tabbar tabs) [#5845](https://github.com/theia-ide/theia/pull/5845)
```

Comment:
I think we should remove reference to `nodes` here, this is likely leftover from the previous rename.